theory-test-recognise-candidate/lambda.js

```diff
@@ -1,5 +1,5 @@
-const logger = require('logger');
-const main = require('./src/main');
+const logger = require('../logger');
+const compareVideoToImage = require('../compare-video-to-image');
```

Comment:
The compare-vedio-to-image is also a module, so just import it by require('compare-video-to-image').